Engines/Wine/QuickScript/Steam Script/script.js

```diff
@@ -119,6 +120,7 @@ SteamScript.prototype.go = function () {
         .prefix(this._name, this._wineDistribution, this._wineArchitecture, this._wineVersion)
         .luna();
     wine.corefonts();
+    wine.DXVK(); // Required for big picture
```

Comment:
While I could agree to add it for the steam app itself, it seems weird to add it here, because then all SteamScript will implicitly use DXVK, and I do not think it is a good idea.

Reply:
We can in theory load DXVK only for tenfoot (Big Picture) or depending on the implementation i was thinking about providing latest minus one version DXVK in phoenicis wine that can be overwritten by script on demand if game depends on different version.

Reply:
Also we need to keep in mind that not everybody will be able to run DXVK.

Reply:
@plata elaborate

Reply:
To quote from the [DXVK README](https://github.com/doitsujin/dxvk):

> Notes on Vulkan drivers
Before reporting an issue, please check the Wiki page on the current driver status and make sure you run a recent enough driver version for your hardware.

> Online multi-player games
Manipulation of Direct3D libraries in multi-player games may be considered cheating and can get your account banned. This may also apply to single-player games with an embedded or dedicated multiplayer portion. Use at your own risk.

So even if we assume that everybody can use Vulkan (which is not the case), there are cases where DXVK cannot be used. I don't want anybody blaming us that his/her account in a multi-player game is banned. Therefore I think we cannot add DXVK by default anywhere (neither for Steam nor for Phoenicis Wine in general).

Reply:
So we have to provide standalone DXVK libs for steam which doesn't affect any other apps.

Reply:
Does it work if you copy dxvk libs next to steam.exe and set those to builtin for steam.exe only so that it does not affect other apps ?

Reply:
I have no clue. Maybe these questions should be raised to the DXVK devs. Or we simply continue to use DXVK on an per app basis only.
There's also one other point which might need consideration: It seems that Wine officially will go for the Wine D3D Vulkan implementation instead of DXVK. So maybe DXVK will become obsolete in the future.

Reply:
Yes, but this is in the "far" future, and the two backend OpenGL/Vulkan will remains.

Reply:
That's true. However, isn't that a reason to use DXVK only per app? It will allow us to always implement what works best.

Reply:
How do you do it is the question I was asking.

Reply:
That's my understanding as well.

Reply:
afaik there is a way to use certain lib for wineapp only, investigating atm.. 

do not bother sujin with this since it's not DXVK question..